doc/source/getting_started/wsl.rst

```diff
@@ -213,8 +218,48 @@ If you want to change the working directory, you can use the ``-dir`` flag.
     /ansys_inc/v222/ansys/bin/ansys222 -grpc -dir /tmp/ansys_jobs/myjob
 
 
+Launch MAPDL in the Windows host OS
+###################################
+
+You can launch an instance of MAPDL using the MAPDL installation from the
+Windows host OS.
+To do that, you can just run the following code.
+
+.. code:: python
+
+   from ansys.mapdl.core import launch_mapdl
+
+   mapdl = launch_mapdl(
+       exec_file="/mnt/c/Program Files/ANSYS Inc/v231/ANSYS/bin/winx64/ANSYS231.exe",
+   )
+
+As mentioned in `Open ports for license server communication`_, the Windows host OS
+and WSL are connected with a virtual network where they have both different IPs.
+PyMAPDL does its best to detect the IP of the Windows host OS. For that, it parses
+the output given by the command `ip route` in WSL.
+However, if you find that this IP is not correct, you can specify the IP to connect
+to using:
+
+.. code:: python
+
+   from ansys.mapdl.core import launch_mapdl
+
+   mapdl = launch_mapdl(
+       exec_file="/mnt/c/Program Files/ANSYS Inc/v231/ANSYS/bin/winx64/ANSYS231.exe",
+       ip="172.23.112.1",
+   )
+
+You might need to disable the Microsoft Firewall completely or at least
+for the WSL network connection.
+To do so, follow `this link <disable_firewall_on_wsl_ethernet_section_>`_.
```

Comment:
Google developer documentation style guide says that you should always use the title of the article, topic, section, or whatever is the link text so that the user knows where the link will take him or her.

Reply:
Does this apply only to external links??

I wrote later:
```rst
.. note::
   If you want to replicate the CICD behaviour or develop from inside a docker container,
   it is highly recommended to use Ubuntu as base operative system. You can find instructions
   to `create your own MAPDL Ubuntu container in here <ref_make_container_>`_ and how to use it to
   `develop on containers in here <ref_devcontainer_>`_.
```

Reply:
Changed there too.